flow360/component/simulation/meshing_param/params.py

```diff
@@ -47,9 +48,6 @@ class MeshingParams(Flow360BaseModel):
     """
 
     # Volume **defaults**:
-    farfield: Optional[Literal["auto", "quasi-3d", "user-defined"]] = pd.Field(
-        default="auto", description="Type of farfield generation."
-    )
```

Comment:
how do we keep this behaviour: default="auto" ? I would prefer user does not need to do it explicitly especially on webUI?

Reply:
We can change this in the get_default_param service?